provisioning/prelude-manager-oss/install-packages.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash -exe
 
-yum install -y \
-  libprelude \
-  mariadb-server \
-  prelude-manager \
-  prelude-manager-db-plugin \
-  prelude-tools \
-  preludedb-mysql \
-  pwgen
+dnf install -y prelude-manager \
```

Comment:
can you keep this package list alphabetically ordered? Also add a new line after -y option, like:
dnf install -y \
package1 \
package2